mdpow/fep.py

```diff
@@ -855,7 +865,7 @@ def contains_corrupted_xvgs(self):
         :attr:`Gsolv._corrupted` as dicts of dicts with the component as
         primary and the lambda as secondary key.
         """
-        from itertools import izip
+        from six.moves import zip
```

Comment:
Do all imports at the top of the file, makes it a lot easier to keep track of what's needed.

The only time imports inside functions are justified if you _must_ delay the import until runtime, eg because it's an optional dependency.

Reply:
```suggestion
```